<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="nodeList">
        <property name="maximumSize">
         <size>
          <width>281</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="button_deleteNode">
          <property name="maximumSize">
           <size>
            <width>88</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string>deleteNode</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="nodeKey">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>28</height>
           </size>
          </property>
          <property name="maxLength">
           <number>32770</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="button_addNode">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string>addNode</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="button_searchNode">
          <property name="text">
           <string>searchNode</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
